tools/make-framework

```diff
@@ -0,0 +1,418 @@
+#!/bin/bash
```

Comment:
I suggest using `-e`, that way errors aren't silently ignored:

```shell
#!/bin/bash -e
```

Reply:
👍  That's in my list of TODOs.
Will look at shellcheck.

Reply:
I put in most of the spellcheck changes. Most of them were quotations

Reply:
This is about as far as this can go without addressing other issues like how it's packaged and where it will live.

Reply:
Not sure I follow: do you mean to say that you can't add `-e` yet, because you'd run into errors you want to ignore?

Reply:
For that, I haven't decided whether I want `-e` and fail hard or if I want to manually handle the errors. This script will be run from both the build and from the internals of BTfS.

Reply:
You can always decide on a case-to-case basis.

One way to ignore failures is to add `|| true` to commands you don't care about:

```shell
some command that may fail || true
```

Or you can print it out:

```shell
if ! some command that may fail; then
    echo "Some command that may fail failed, but we don't care."
fi
```